package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "appium.io",
+  "version": "1.0.0",
+  "description": "Documentation project for Appium.io",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "serve": "bundle exec jekyll serve && npm run build:docs",
+    "build:docs": "./node_modules/babel-cli/bin/babel-node.js scripts/repo.js",
```

Comment:
The `./node_modules/...` shouldn't be necessary. npm should copy all that into `./node_modules/.bin` and then add that to the beginning of the PATH when using `npm run script`.